src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -73,6 +76,18 @@ public synchronized void addPerson(ReadOnlyPerson person) throws DuplicatePerson
     }
 
     @Override
+    public void removeTag(Tag tag) throws PersonNotFoundException, DuplicatePersonException {
+        for (int i = 0; i < addressBook.getPersonList().size(); i++) {
```

Comment:
Seems like the `getPersonList()` was used more than once - would you want to store the person list in a variable instead? It might make it more readable.